pkg/streamingpromql/engine_test.go

```diff
@@ -2337,6 +2364,29 @@ func TestRateIncreaseAnnotations(t *testing.T) {
 			expectedInfoAnnotations:    []string{},
 		}
 	}
+}
+
+func TestDeltaAnnotations(t *testing.T) {
+	nativeHistogramsWithGaugeResetHints := `
+		metric{series="mix-float-nh"} 10 {{schema:-53 sum:1 count:1 custom_values:[5 10] buckets:[1] counter_reset_hint:gauge}} {{schema:-53 sum:1 count:1 custom_values:[5 10] buckets:[1] counter_reset_hint:gauge}} {{schema:-53 sum:5 count:4 custom_values:[5 10] buckets:[1] counter_reset_hint:gauge}}
+		metric{series="mixed-exponential-custom-buckets"} {{schema:0 sum:1 count:1 buckets:[1]}} {{schema:-53 sum:1 count:1 custom_values:[5 10] buckets:[1]}} {{schema:0 sum:5 count:4 buckets:[1 2 1]}}
+	`
+
+	testCases := map[string]annotationTestCase{}
```

Comment:
[nit] Given the test case names are known statically (unlike `TestRateIncreaseAnnotations`), this could be written something like this:

```golang
testCases := map[string]annotationTestCase{
    "delta() over series...": {
        ...
    },
}
```

Reply:
Updated in 617cfeb9872a90178f8ee5df0cd63544243ba84c